tests/unit/s2n_server_hello_test.c

```diff
@@ -455,7 +504,8 @@ int main(int argc, char **argv)
         /* the client only support TLS1.2 */
         EXPECT_SUCCESS(s2n_enable_tls13());
         server_conn->actual_protocol_version = S2N_TLS12;
-        server_conn->secure.cipher_suite = &s2n_ecdhe_ecdsa_with_aes_128_gcm_sha256;
+        /* The server must provide a cipher which is in the client's preference list */
```

Comment:
Do we need these comments everywhere? They make sense in the context of this PR, but I'm not sure they're helpful long term. Especially since values that were right before won't have them.

Reply:
Makes sense, removed.